.jenkins/continuous.groovy

```diff
@@ -271,23 +271,26 @@ pipeline {
                               cmake \
                                 -DCMAKE_BUILD_TYPE=Release \
                                 -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
-                                -DCMAKE_CXX_COMPILER=clang++ \
-                                -DCMAKE_CXX_FLAGS="-fsycl-device-code-split=per_kernel -Wno-deprecated-declarations -Werror -Wno-gnu-zero-variadic-macro-arguments -Wno-unknown-cuda-version -Wno-sycl-target" \
-                                -DCMAKE_PREFIX_PATH="$ONE_DPL_DIR" \
+                                -DCMAKE_CXX_COMPILER=icpx \
+                                -DCMAKE_CXX_FLAGS="-fsycl-device-code-split=per_kernel -fp-model=precise -Wno-deprecated-declarations -Werror -Wno-gnu-zero-variadic-macro-arguments -Wno-unknown-cuda-version -Wno-sycl-target" \
                                 -DKOKKOS_IMPL_SYCL_DEVICE_GLOBAL_SUPPORTED=0 \
+                                -DKOKKOS_IMPL_HAVE_SYCL_EXT_ONEAPI_DEVICE_GLOBAL=OFF \
```

Comment:
This is naughty.
Do I understand it right that this overrides our detecting existence of the `SYCL_EXT_ONEAPI_DEVICE_GLOBAL` symbol?

Reply:
Yes, everything compiles fine but creates runtime errors for NVIDIA and HIP GPUs.

Reply:
Sure, we can revisit later what the issue for non-Intel GPUs is.